src/renderer/i18n/nl.js

```diff
@@ -0,0 +1,483 @@
+    loadDefaultSuccess: "Default ingeladen!",
+    copyToClipboard: "Kopieer naar klipbord",
+    copySuccess: "Gekopieerd!",
+    pasteFromClipboard: "Plakken van klipbord",
+    pasteSuccess: "Geplakt!",
+    onlyCustom: {
+      warning: `Chrysalis ondersteunt niet langer configuraties die een mix van hardcoded and EEPROM lagen bevat. Als dit een feature is die je nodig hebt, dan horen we graag over jouw use case. In de meeste gevallen raden we echter aan om over te schakelen naar enkel aangepaste lagen, wat Chrysalis voor jou kan doen. Wanneer je de switch maakt zullen hardcoded lagen niet gebruikt worden, en de standaard laag die gezet wordt (als er een gezet wordt) is laag nul.`,
+      fixItButton: "Verander naar enkel aangepaste lagen",
+      openFR: "Open een feature request",
+    },
+    macros: {
+      title: "Macro #{{ index }}",
+      edit: "Macro aanpassen",
+      out_of_space: `Dynamische macros nemen te veel ruimte in, maak aub {{ overflow }} bytes vrij.`,
+      test: {
+        button: "Test",
+        placeholder: "Klik 'Test', en de macro zal hier afspelen.",
+      },
+      steps: {
+        time_ms: "{{ value }}ms",
+        in_ms: "ms",
+        unknown: "<onbekend>",
+        INTERVAL: "Vertraging tussen stappen",
+        WAIT: "Wacht",
+        KEYDOWN: "Hou toets ingedrukt",
+        KEYUP: "Laat toets los",
+        TAP: "Toets kort indrukken",
+        EXPLICIT_REPORT: "Zend geen rapport tussen stappen",
+        IMPLICIT_REPORT: "Zend een rapport tussen stappen",
+        SEND_REPORT: "Zend nu een rapport",
+        unsupported: "Niet ondersteund",
+      },
+    },
+  },
+  preferences: {
+    interface: "Gebruikersinterface",
+    advanced: "Geavanceerd",
+    verboseFocus: "Verbose logging",
+    devtools: {
+      main: {
+        label: "Ontwikkelaars tools",
+        description: `Deze voorkeuren helpen voornamelijk ontwikkelaars. Het kan zijn dat je gevraagd wordt om deze aan te zetten om meer informatie te kunnen verzamelen over een fout.`,
+      },
+      verboseLogging: {
+        label: "Verbose logging aanzetten",
+        help: "Zet een goede hoeveelheid extra logging aan, handig tijdens het debuggen.",
+      },
+      console: {
+        label: "Open de ontwikkelaarsconsole",
+        help: "De ontwikkelaarsconsole laat je kijken binnenin Chrysalis. Pas op met het gebruik hiervan.",
+      },
+    },
+    ui: {
+      layoutEditor: {
+        label: "Layout Editor aanpassen",
+      },
+      layoutCards: {
+        label: "Layout Kaarten aanpassen",
+      },
+      hideUnavailableFeatures: {
+        label: `Verberg features die niet worden ondersteund door de huidige firmware van je toetsenbord`,
+        help: `Wanneer aangezet zal Chrysalis configuratieopties verbergen van features die niet ondersteund worden door de huidige firmware van je toetsenbord.`,
+      },
+      lookNFeel: {
+        label: "Pas uitzicht aan",
+      },
+      theme: {
+        label: "Algemeen uitzicht",
```

Comment:
`uitzicht` refers to things you see or the future, I think you meant `uitstraling`?

```suggestion
      lookNFeel: {
        label: "Pas uitstraling aan",
      },
      theme: {
        label: "Algemene uitstraling",
```

Reply:
I'll change this to "Pas uiterlijk aan" and "Algemeen uiterlijk"